### PR TITLE
Fix: onChange eventbus

### PR DESCRIPTION
**Pull request by jung-han**
### Please check if the PR fulfills these requirements
- [ ] It's submitted to right branch according to our branching model
- [ ] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: src/view/editingLayerInner.tsx [hunk 1]
```diff
@@ -90,6 +90,13 @@ export class EditingLayerInnerComp extends Component<Props> {
       this.contentEl.appendChild(editorEl);
       this.editor = cellEditor;
 
+      const editorWidth = (this.editor.el as HTMLElement).getBoundingClientRect().width;
+
+      if (editorWidth > width) {
+        // add cell padding width
+        (this.contentEl as HTMLElement).style.width = `${editorWidth + 10}px`;
```
Comment (seonim-ryu) [inline src/view/editingLayerInner.tsx]: `10`은.. 셀 패딩값 때문에 추가되는건가요?ㅁ?

Comment (jung-han) [reply src/view/editingLayerInner.tsx]: 넵

File: src/view/editingLayerInner.tsx [hunk 2]
```diff
@@ -90,6 +90,13 @@ export class EditingLayerInnerComp extends Component<Props> {
...
+      }
```
Comment (js87zz) [inline src/view/editingLayerInner.tsx]: `10px`은 따로 상수에 할당하고 하는게 더 좋을 것 같습니다!

File: src/view/editingLayerInner.tsx (hunk 1 above)
Comment (seonim-ryu) [inline src/view/editingLayerInner.tsx]: `10`은.. 셀 패딩값 때문에 추가되는건가요?ㅁ?

**Comment by js87zz** [comment]
리뷰 완료입니다!

**Pull request merged**